tst/testinstall/strings.tst

```diff
@@ -262,6 +262,12 @@ gap> fname := Filename(tmpdir, "output.csv");;
 gap> PrintCSV( fname, s );
 gap> s = ReadCSV( fname );
 true
+gap> s := ReadCSV( fname, true );
```

Comment:
This new test actually does not use RCSVSplitString. To do that, the input must contain quotes " and use commas as sepetators. The quotes then are used to mask fields containing commas.

Reply:
@fingolfin 
You're right. We need to force commas in strings, and maybe lets add some gratuitous quotes. But it does not need to be the mask.

What about

    gap> s[2].emil:="\"Alas, poor Yorick\", the call went";;
    gap> PrintCSV(fnam,s);
    gap> ReadCSV(fnam,true);
    [ rec( field1 := "emil", field2 := "f", field3 := "f1", field4 := "f 2" ),
      rec( field2 := 2, field3 := 1, field4 := 3 ),
      rec( field1 := "\"Alas, poor Yorick\", the call went", field3 := 4, field4 := 6 ),
      rec( field3 := 7 ),  rec( field2 := 11, field4 := 12 ),
      rec( field2 := "yy", field3 := "x", field4 := "zzz" ) ]